server/commands/userCreator.ts

```diff
@@ -1,12 +1,18 @@
 import { sequelize } from "@server/database/sequelize";
 import InviteAcceptedEmail from "@server/emails/templates/InviteAcceptedEmail";
 import { DomainNotAllowedError, InviteRequiredError } from "@server/errors";
-import { Event, Team, User, UserAuthentication } from "@server/models";
+import {
+  Event,
+  Team,
+  User,
+  UserAuthentication,
+  AuthenticationProvider,
+} from "@server/models";
 
 type UserCreatorResult = {
   user: User;
   isNewUser: boolean;
-  authentication: UserAuthentication;
+  authentication: UserAuthentication | null;
```

Comment:
authentication can now return null, for cases where there is a login through simple email matching. this has no practical impact, as this part of the result was never used